main.py

```diff
@@ -126,6 +126,16 @@ def execute_consistency(self):
             stored_circuits.pop(circuit.id, None)
             if self.should_be_checked(circuit):
                 circuits_to_check.append(circuit)
+            # setup failover_path whenever possible
+            if getattr(circuit, "failover_path", None):
+                continue
+            affected_at = getattr(circuit, "affected_by_link_at", None)
```

Comment:
We should have ways to expedite the failover setup if not Link events affected the EVC recently created. This can be an improvement for future

Reply:
Fixed on commit 9fb511b